lib/drm/drm_engine.js

```diff
@@ -427,6 +427,14 @@ shaka.drm.DrmEngine = class {
             (this.config_.advanced &&
             this.config_.advanced[drmInfo.keySystem] &&
             this.config_.advanced[drmInfo.keySystem][robustnessType]) || '';
+        if (items == '' &&
```

Comment:
I guess it's out of scope for this PR, but I probably should have asked for this `expandRobustness` function to have JSDoc for the parameters and return type...

Reply:
Done!